pyproject.toml

```
@@ -33,10 +33,10 @@ classifiers = [
 # `ci/environment-mindeps.yaml` conda environment. When updating minimum dependencies
 # here, make sure to also update `ci/environment-mindeps.yaml`.
 [tool.poetry.dependencies]
-python = ">=3.8,<4.0"
+python = ">=3.8"
```

```suggestion
python = ">=3.8,<4.0"
```
Looks like this is actually problematic with poetry and cannot be relaxed :roll_eyes: 
* https://github.com/zanieb/poetry-relax?tab=readme-ov-file#frequently-asked-questions
* https://github.com/orgs/python-poetry/discussions/3757#discussioncomment-435345